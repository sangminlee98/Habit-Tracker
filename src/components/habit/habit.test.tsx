import { render, screen } from "@testing-library/react";
import renderer from "react-test-renderer";
import Habit from "./habit";

describe("Habit", () => {
  it("renders", () => {
    const component = renderer.create(
      <Habit
        habit={{ id: 1, name: "test", count: 1 }}
        onIncrement={jest.fn()}
        onDecrement={jest.fn()}
        onDelete={jest.fn()}
      />
    );
    expect(component).toMatchSnapshot();
  });
  describe("Habit Test", () => {});
});
