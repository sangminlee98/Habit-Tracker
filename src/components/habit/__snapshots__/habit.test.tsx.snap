// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Habit renders 1`] = `
<li
  className="habit"
>
  <span
    className="habit-name"
  >
    test
  </span>
  <span
    className="habit-count"
  >
    1
  </span>
  <button
    className="habit-button habit-increase"
    onClick={[Function]}
  >
    <i
      className="fas fa-plus-square"
    />
  </button>
  <button
    className="habit-button habit-decrease"
    onClick={[Function]}
  >
    <i
      className="fas fa-minus-square"
    />
  </button>
  <button
    className="habit-button habit-delete"
    onClick={[Function]}
  >
    <i
      className="fas fa-trash"
    />
  </button>
</li>
`;
